src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs

```diff
@@ -231,9 +231,13 @@ public AdjustmentRule[] GetAdjustmentRules()
             if (GlobalizationMode.Invariant)
                 return displayName;
 
+#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
+            return displayName;
+#else
```

Comment:
aren't we skipping the new native code completely if we return before calling GetFullValueForDisplayNameField here?

Reply:
Updated to use `NSTimeZone.name` for hybrid mode